File: Python/UberVan.py
from car import Car

class UberVan(Car):
  typeCarAccepted = []
  seatsMaterials = []

  def __init__(self, license, driver, typeCarAccepted, seatsMaterials):
    super.__init__(license, driver)
    self.typeCarAccepted = typeCarAccepted
    self.seatsMaterials = seatsMaterials

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
  Map<String, Map<String, Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterials;

  public UberVan(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted,
      ArrayList<String> seatsMaterials) {
    super(license, driver);
    this.typeCarAccepted = typeCarAccepted;
    this.seatsMaterials = seatsMaterials;
  }
}
